src/test_heap.py

```diff
@@ -1,12 +1,52 @@
 # -*- coding: utf-8 -*-
+import pytest
 
 
-TEST_LIST_ONE = [40, 33, 60, 19, 10]
-TEST_LIST_TWO = [34, 56, 78, 90, 3, 4, 1, 6, 29, 66, 5, 4, 8, 9, 24, 20]
-TEST_LIST_THREE = [24, 45, 92, 93, 65, 87, 19, 9, 46, 72, 85, 4, 76, 900]
+TEST_LISTS = ([40, 33, 60, 19, 10],
+              [34, 56, 78, 90, 9, 24, 20, 3, 4, 1, 6, 29, 66, 5, 4, 8],
+              [24, 45, 92, 93, 65, 87, 19, 9, 46, 72, 85, 4, 76, 900],
+              [34, 56, 78, 90, 3, 4, 1, 6, 29, 66, 5, 4, 8, 9, 24, 20, 24, 45, 92, 93, 65, 87, 19, 9, 46, 72, 85, 4, 76, 900])
```

Comment:
Remember to test edge cases. What happens when your list is only 1 node? What about an empty list?